### ISSUE TITLE
Unusual loss behavior

### ISSUE THREAD

**Issue by sakvaua** [mlyg/DicePlusPlus#2]
Hi!
Thanks for your loss (sounds weird).
I tried applying to a 3D Unet style CNN and got this unusually looking validation loss with a sharp drop to a different level. There was no adjustment to the learning rate. I don't have checkpoints from this moment, so I'm not sure what happened. The dice that I monitor (but don't really care as my metric is non-differentiable and quite different) also went down, so loss improved significantly, but the dice became worse. I wonder if you have experienced such behavior and know what causes it.

![Image](https://github.com/user-attachments/assets/857fe9c5-1d08-4b04-a78f-8f5ffa7b9ada)

Oh, and the train loss looks pretty much the same.
Thanks again!
